<launch>
	<param name="use_sim_time" value="true"/>
	<include file="$(find privacy_zones)/launch/zone_sim.launch"/>
	<include file="$(find peac_bridge)/launch/local_demo.launch"/>
	<node name="light_follower" pkg="privacy_zones" type="lightFollower.py"/>
</launch>